modules/ocf_mirrors/manifests/qt.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_mirrors::qt {
+  file {
+    '/opt/mirrors/project/qt':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/qt/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  cron {
+    'qt':
+      command => '/opt/mirrors/project/qt/sync-archive > /dev/null',
+      user    => 'mirrors',
+      hour    => '*/4',
+      minute  => '5';
```

Comment:
Maybe we could do some more automated spacing out of these with `fqdn_rand(60, $mirror_name)` instead of manually assigning them? It would do a better job spacing these out then we would and then adding new ones would be even easier.